src/io/flutter/FlutterBundle.properties

```diff
@@ -7,6 +7,12 @@ app.reload.action.description=Hot reload changes into the running Flutter app
 app.restart.action.text=Flutter Full Restart
 app.restart.action.description=Restart the Flutter app
 
+app.profile.action.text=Run in profile mode...
+app.profile.action.description=Run Flutter app in profile mode...
```

Comment:
We could even include the word 'Flutter' in the action name (not just the description)? It would be more specific, if a bit long.